pom.xml

```diff
@@ -208,7 +208,7 @@
 					<systemPropertyVariables>
 						<test.build.data>${project.basedir}/target/test-data/</test.build.data>
 					</systemPropertyVariables>
-					<argLine>@{argLine} -Xmx512m</argLine>
```

Comment:
Why was this commented out? Just curious.

Reply:
My mistake, erroneously committed. I'll add it again. Thanks! Eclipse fails to run the unit tests and complains that it does not find `argLine`.